src/locales/default.json

```diff
@@ -293,6 +293,9 @@
     "confirmLeaveMessage": " Do you really want to leave? You have unsaved changes that will be lost.",
     "confirmDeleteSpace": " Do you really want to delete this space? This action cannot be undone.",
     "confirmInputDeleteSpace": "Enter {space} to continue:",
+    "noticeSettingsSavedTitle": "Settings saved",
```

Comment:
We can remove settings from the naming as it is already nested inside the settings object/